sklearn/model_selection/tests/test_split.py

```diff
@@ -664,8 +664,32 @@ def test_stratified_shuffle_split_overlap_train_test_bug():
 
     train, test = next(iter(sss.split(X=X, y=y)))
 
+    # no overlap
     assert_array_equal(np.intersect1d(train, test), [])
 
+    # complete partition
+    assert_array_equal(np.union1d(train, test), np.arange(len(y)))
```

Comment:
I strengthened this unrelated test as well.  (Here because of the high number of classes it's not trivial to check test proportions)